wiki_articles/endl.md

```diff
@@ -1,28 +1,35 @@
 # About `std::endl`, Buffers, And Flushing
 
-I/O in C++ (`std::cout`, i.e.`std::ostream`) is *buffered*,
-which means that not every byte you write to the stream is instantly written through to the terminal/disk.
+I/O in C++
+(**[std::cout](https://en.cppreference.com/w/cpp/io/cout)**, i.e.
+**[std::ostream](https://en.cppreference.com/w/cpp/io/basic_ostream)**)
+is *buffered*, which means that not every byte you write to the stream is
```

Comment:
It might be more clear to beginners to just say character instead of a byte here